packages/calcite-components/src/utils/sortableComponent.ts

```diff
@@ -12,27 +18,100 @@ export interface SortableComponent {
    */
   readonly el: HTMLElement;
 
+  /**
+   * When `true`, dragging is enabled.
+   */
+  dragEnabled: boolean;
+
+  /**
+   * Specifies which items inside the element should be draggable.
+   */
+  dragSelector?: string;
+
+  /**
+   * The list's group identifier.
+   */
+  group?: string;
+
+  /**
+   * The selector for the handle elements.
+   */
+  handleSelector: string;
+
   /**
    * The Sortable instance.
    */
   sortable: Sortable;
+
+  /**
+   * Whether the element can  move from the list.
+   */
+  dragCanPull: (event: DragEvent) => boolean;
```

Comment:
In order to simplify the API a bit, would `canPull` and `canPut` be concise and expressive enough for these props?

Reply:
Yeah I can make that change